src/constants.ts

```diff
@@ -17,6 +17,15 @@ export const FEATURE_REQUEST_URL =
 	'https://github.com/Automattic/studio/issues/new?assignees=&labels=%5BType%5D+Feature+Request&projects=&template=feature_request.yml&title=Feature+Request%3A';
 export const WPCOM_PROFILE_URL = `https://wordpress.com/me`;
 
+//Import file constants
+
+export const ACCEPTED_FILE_TYPES = [
```

Comment:
Nit: what do you think about renaming the variable itself and removing the comment?
```suggestion
//Import file constants

export const ACCEPTED_IMPORT_FILE_TYPES = [
```

Or maybe ACCEPTED_BACKUP_FILE_TYPES

Reply:
Let's use `ACCEPTED_IMPORT_FILE_TYPES` - I think this verbiage is clear for the purpose of it 👍  
Updated in https://github.com/Automattic/studio/pull/378/commits/c3baca777a7beaf1623779fbb44fc5aa6a0bc72d